dhall/src/Dhall/Main.hs

```diff
@@ -440,7 +454,9 @@ command (Options {..}) = do
                         then Annot alphaNormalizedExpression inferredType
                         else alphaNormalizedExpression
 
-            render System.IO.stdout annotatedExpression
+            case output of
+                StandardOutput -> render System.IO.stdout annotatedExpression
+                Output file_ -> System.IO.withFile file_ System.IO.WriteMode $ (flip render) annotatedExpression
```

Comment:
Can you move the code after `->` to the next line and write it without `flip`? The easier to read the better!

Reply:
Regarding the `flip`, sure. But it will also caused several changes to other lines. Is it okay?

Reply:
I'm not quite sure what you mean, but please give it a try.